nominal/core/connection.py

```diff
@@ -198,6 +211,12 @@ def unarchive(self) -> None:
         self._clients.connection.unarchive_connection(self._clients.auth_header, self.rid)
 
 
+def _get_connections(
+    clients: Connection._Clients, connection_rids: Iterable[str]
+) -> Iterable[scout_datasource_connection_api.Connection]:
```

Comment:
nit: a list is an iterable but I'd probably just either annotate this as a sequence or return a generator expression by swapping the [] for ()